papaparse.js

```diff
@@ -167,6 +168,42 @@
 		}
 	}
 
+	// Ability to parse file(s) when using with Node
+	function ParseFiles(_input, _config)
+	{
+		if (Array.isArray(_input)) {
+			var results = [];
+			_input.forEach(function(input) {
+				if(typeof input === 'object')
+					results.push(ParseFiles(input.file, input.config));
+				else
+					results.push(ParseFiles(input, _config));
+			});
+			return results;
+		} else {
+			var results = {
+				data: [],
+				errors: []
+			};
+			if ((/(\.csv|\.txt)$/).test(_input)) {
```

Comment:
Let's not restrict the file extension - there could be any number of different valid ones, or none at all.